DependencyInjection/DoctrineExtension.php

```diff
@@ -929,7 +929,7 @@ private function createMetadataCache(string $objectManagerName, ContainerBuilder
 
         $cache = new Definition(ArrayAdapter::class);
 
-        if (! $container->getParameter('kernel.debug')) {
+        if (! $entityManager['cache_metadata']) {
```

Comment:
:thinking: shouldn't it be the opposite?
```suggestion
        if ($entityManager['enable_metadata_cache']) {
```

Reply:
Fixed it. I'll add some tests!